ecrupdater.py

```diff
@@ -4,24 +4,42 @@
 import time
 import base64
 import boto3
+import logging
+import logging.config
 from kubernetes import client as k8sclient
-from kubernetes.client import Configuration, ApiClient
+from kubernetes.client import Configuration, ApiClient, ApiException
+
+
+def setup_logging(
+        log_config_file=os.environ.get('LOG_CONFIG', './logging.json'),
+        # fallback if no logger config file has been configured
+        default_level=os.environ.get('LOG_LEVEL', 'INFO')
+):
+    if os.path.exists(log_config_file):
+        print("Loading log configuration file from path '{}'...".format(log_config_file))
+        with open(log_config_file, 'rt') as f:
+            config = json.load(f)
+        logging.config.dictConfig(config)
+    else:
+        print("No Log config file provided, loading default configuration.")
+        logging.basicConfig(level=default_level)
 
 
 def exception_handler(exception_type, exception, traceback):
     # Removes standard python stacktrace in case of exceptions
```

Comment:
Is this a wanted behaviour to remove the stacktrace? (If yes LGTM)

Reply:
I don't know why they want to do this actually..

Reply:
OK well, it was like this before so 🤷